<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadBookmarkDialog</class>
 <widget class="QDialog" name="LoadBookmarkDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load Bookmark</string>
  </property>
  <layout class="QVBoxLayout" name="topLayout" stretch="0,0,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="XmlUi::GradientWidget" name="promptWidget" native="true">
     <layout class="QHBoxLayout" name="promptLayout" stretch="0,1">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="promptPixmap"/>
      </item>
      <item>
       <widget class="QLabel" name="promptLabel">
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="mainWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>9</number>
      </property>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>9</number>
        </property>
        <item>
         <widget class="BookmarkListView" name="listView"/>
        </item>
        <item>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QGroupBox" name="groupPreview">
            <property name="title">
             <string>Preview</string>
            </property>
            <layout class="QVBoxLayout">
             <item>
              <widget class="ViewContainer" name="viewContainer" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XmlUi::GradientWidget</class>
   <extends>QWidget</extends>
   <header>xmlui/gradientwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ViewContainer</class>
   <extends>QWidget</extends>
   <header>viewcontainer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BookmarkListView</class>
   <extends>QListView</extends>
   <header>bookmarklistview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LoadBookmarkDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LoadBookmarkDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
